app/helpers/bgeigie_imports_helper.rb

```diff
@@ -46,6 +46,10 @@ def moderator?(user)
     user.try!(:moderator?)
   end
 
+  def approval_participant?(user, bgeigie_import)
+    user.moderator? || user == bgeigie_import.user
+  end
```

Comment:
I think we can just use `current_user` helper inside
```suggestion
  def approval_participant?(bgeigie_import)
    current_user.moderator? || current_user == bgeigie_import.user
  end
```